wazo_dird_plugin_odoo/api.yml

```diff
@@ -121,6 +121,16 @@ definitions:
             example: "ip_of_odoo_server"
             default: ""
             type: string
+          model:
```

Comment:
When you add new definitions in spec you need to update the schemas.py to reflect this changes. If you don't it can work.